railgun/Makefile

```diff
@@ -1,11 +1,26 @@
-KIND_CLUSTER_NAME ?= "integration-tests"
+##@ Versioning
+
+TAG ?= 2.0.0-alpha.0
```

Comment:
This tag:
- will be an extra burden to keep up to date
- is of questionable value to me and may end up as noise, because our users are going to use to Docker build which injects the tag based on the Git tag automatically; this will effectively only be in use by custom builds and development, which is almost guaranteed to run modified versions, not matching the tag exactly.

Suggestion: remove the default value of `TAG`

Reply:
Sounds good 👍